woocommerce/woocommerce
# Show linkout to Marketplace when there are no product metabox suggestions to display

### Changes proposed in this Pull Request:
When a user has dismissed all the product metabox suggestions, or if there are none to display, the UI is a little confusing.

This PR shows a nice message and button linking to the marketplace so it's clear why it's there.

### How to test the changes in this Pull Request:
1. Clone this branch and get it running as your woocommerce extension in a test environment.
2. Edit a product, scroll down to metabox, click `Get more options` tab.
3. Dismiss all the suggestions – should see nice new UI.

Also, after dismissing all the suggestions, reload the page, and click on the tab.

### Screenshots

<img width="943" alt="Screen Shot 2019-03-13 at 9 50 23 AM" src="https://user-images.githubusercontent.com/4167300/54235139-7781db80-4575-11e9-99d6-e2981f1a9ffc.png">
